app/models/robot_world.rb

```diff
@@ -1,56 +1,80 @@
+require_relative 'statbot'
 class RobotWorld
-  attr_reader :world
-  def initialize(world)
-    @world = world
+  include StatBot
+  attr_reader :database, :robot_ages, 
```

Comment:
since a few of these methods don't even live in this class specifically (they live in `StatBot`), i wonder if it makes sense to not have those `attr_reader`'s live in this class. it's pretty confusing to read and follow as it is